docs/sql/scalar.txt

```diff
@@ -835,9 +835,132 @@ Examples
     +---------------------+--------------+
     SELECT 5 rows in set (... sec)
 
+
+Array Functions
+===============
+
+array_cat(first_array, second_array, [ third_array , ... ]) returns array
+-------------------------------------------------------------------------
+
+The ``array_cat`` function concatenates one or more arrays into one array
+
+::
+
+    cr> select array_cat([1,2,3],[3,4,5,6],[7,9,10]) from sys.cluster;
+    +-----------------------------------------+
+    | array_cat([1,2,3], [3,4,5,6], [7,9,10]) |
+    +-----------------------------------------+
+    | [1, 2, 3, 3, 4, 5, 6, 7, 9, 10]         |
+    +-----------------------------------------+
+    SELECT 1 row in set (... sec)
+
+
+It can be used to append elements to array fields
+
+::
+
+    cr> create table array_cat_example (list array(integer));
+    CREATE OK (... sec)
+
+::
+
+    cr> insert into array_cat_example (list) values ([1,2,3]);
+    INSERT OK, 1 row affected (... sec)
+
+::
+
+    cr> update array_cat_example set list = array_cat(list, cast([4, 5, 6] as array(integer)));
```

Comment:
this statement may fail (see travis). 
you need to do a ``refresh table array_cat_example;`` before issueing the update here.
in fact you need to do one after the update as well in to ensure that the select below will get the updated value.